seb_openedx/utils.py

```diff
@@ -0,0 +1,22 @@
+""" utils """
```

Comment:
If we do the refactor for get_chapter_from_location, this whole file can go and use the original edx-platform/openedx/core/lib/url_utils.py functions

Reply:
Ok